fix_import.py

```
@@ -0,0 +1,14 @@
+import importlib.util
+import os
+
+filename = [f for f in os.listdir('.') if f.startswith('𝐎𝐧𝐞')][0]
```

Ruff (0.15.17)

[warning] Prefer `next(f for f in os.listdir('.') if f.startswith('𝐎𝐧𝐞'))` over single element slice

Replace with `next(f for f in os.listdir('.') if f.startswith('𝐎𝐧𝐞'))`

(RUF015)